public/components/authentication/authentication.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react'
+import loadingAnimation from '../../images/loading_animation.gif'
+import {
+  RequestUserLoginAuthentication,
+  RequestUserRegisterAuthentication,
+  AuthenticateUserInfo,
+  ResultStatus,
+} from '../requestService/authRequestService'
+import './authentication.css'
+
+const loginText = 'Sign In'
+const registerText = 'Sign Up'
+const toggleToLoginText = 'Already Have An Account'
+const toggleToRegisterText = 'Create An Account'
+
+/**
+ * @param {{
+ * onSuccess: (message: String) => void
+ * onError: (message: String) => void
+ * }} props
+ */
```

Comment:
Is this commented out? If this is no longer used, you should remove it; I'm unfamiliar with how this works, so if this is by design you can drop this comment

Reply:
Oh yeah. These are sorta markdown documentation. It also helps vscode decipher datatypes since javascript dynamic typing is the code equivalent of pure chaos.